internal/telemetry/default.go

```diff
@@ -77,3 +152,33 @@ func (t *defaultTelemetry) getLatestVersion() (interface{}, error) {
 
 	return string(body), nil
 }
+
+func (t *defaultTelemetry) sendRecordedTelemetry() {
+	// There is the possibility of dropping data since we are not atomically
```

Comment:
not great, b/c this will create noise in the race detector output

Reply:
Fixed